src/Aggregate/AggregateChanged.php

```diff
@@ -75,8 +81,8 @@ public function recordNow(int $playhead): self
      */
     public static function deserialize(array $data): self
     {
-        /** @var string $class */
         $class = $data['event'];
+        assert(is_string($class));
```

Comment:
could we use webmozart/assert ?

Reply:
the phpcs fixer did that, I have now solved it with an exception. i don't want to add a library for one check.